src/hooks/useUsers/useUsers.js

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from 'react-query';
+
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+import { useEffect, useState } from 'react';
+
+export const useUsers = (tenant = '') => {
```

Comment:
It is better if you use this approach https://github.com/folio-org/stripes-authorization-components/pull/13/files#diff-9d863bb52798f6b68df28b46ee3e20ed4b3bcb021a5fa8ae5c65831563bb9ba5R29

Reply:
fixed